docs/getting-started-with-micro/index.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Setting up Snowplow Micro locally"
```

Comment:
Should we add something to the page about expected time to setup and test some events?